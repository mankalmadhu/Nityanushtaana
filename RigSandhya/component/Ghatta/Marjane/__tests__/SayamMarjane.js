import React from 'react';
import ಸಾಯ೦ಮಾರ್ಜನೆ from '../SayamMarjane';
import Renderer from 'react-test-renderer';

jest.mock('../Marjane', () => 'ಮಾರ್ಜನೆ' );
jest.mock('../SayamAbhimantrane', () => 'ಸಾಯ೦ಅಭಿಮ೦ತ್ರಣೆ' );
jest.mock('../MarjaneSankalpa', () => 'ಮಾರ್ಜನೆಸ೦ಕಲ್ಪ' );

test("ಪ್ರಾತಃಮಾರ್ಜನೆ", () => {
  expect(Renderer.create(
    <ಸಾಯ೦ಮಾರ್ಜನೆ />
  ).toJSON()).toMatchSnapshot();

});