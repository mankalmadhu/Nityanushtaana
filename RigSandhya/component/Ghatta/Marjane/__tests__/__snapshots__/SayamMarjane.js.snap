// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪ್ರಾತಃಮಾರ್ಜನೆ 1`] = `
<View>
  <ಮಾರ್ಜನೆಸ೦ಕಲ್ಪ />
  <ಮಾರ್ಜನೆ />
  <ಸಾಯ೦ಅಭಿಮ೦ತ್ರಣೆ />
</View>
`;
